Source/WebCore/dom/Element.h

```diff
@@ -35,6 +35,7 @@
 #include "ScrollTypes.h"
 #include "ShadowRootMode.h"
 #include "SimulatedClickOptions.h"
+#include "TrustedType.h"
```

Comment:
I don't know that you can for sure do it, but you should try to use a forward declaration instead.

just doing in the block around line 50.
```
class TrustedHTML;
class TrustedScript;
class TrustedScriptURL;
```

Reply:
If you can't do a forward declaration you at the very least wanna change this to just import the type headers directly. And import TrustedType.h inside of Element.cpp